include/swift/Frontend/FrontendOptions.h

```diff
@@ -262,6 +263,12 @@ class FrontendOptions {
   /// The install_name to use in the TBD file.
   std::string TBDInstallName;
 
+  // The current project version to use in the generated TBD file.
+  version::Version TBDCurrentVersion = {1, 0, 0};
+
+  // The dylib compatibility-version to use in the generated TBD file.
+  version::Version TBDCompatibilityVersion = {1, 0, 0};
```

Comment:
I think we're going to want to just prepopulate TBDGenOptions soon, like we do with IRGenOptions, rather than pass everything through FrontendOptions. What do you think? (Probably for a follow-up, not here.)

Reply:
Yeah, that sounds good. I'll submit that cleanup in a follow-up patch.